test/integration/targets/module_utils/runme.sh

```diff
@@ -4,5 +4,36 @@ set -eux
 
 ANSIBLE_ROLES_PATH=../ ansible-playbook module_utils_basic_setcwd.yml -i ../../inventory "$@"
 
+cleanup() {
+    rm -fv /tmp/nulltest
```

Comment:
Why does the test need to use a path outside the temp work dir provided by ansible-test?

Reply:
It doesn't, will fix.